src/main/java/ItemData.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * ItemData class that has information of what one Item object has as attributes and methods.
+ */
+public class ItemData {
+
+    private final String itemname; // Name of the item
+
+    private final String itemattrib; // The Attribute the item affects
+
+    private final int itemvalue; // The value of the item's effect
```

Comment:
```suggestion
    private final int value; // The value of the item's effect
```

Reply:
And please remember to adjust the names of the variables accordingly in the constructor.